app/page/info/index.tsx

```diff
@@ -67,57 +74,86 @@ export function AccountInfo() {
   const account = useAccount()
 
   const [infoNavigation, setInfoNavigation] = useState<InfoNavigation>(
-    InfoNavigation.Activity
+    InfoNavigation.Tokens
   )
 
   const handleInfoNaviChange = (page: InfoNavigation) => {
     setInfoNavigation(page)
   }
 
   return (
-    <div>
-      {/* Display the Account address */}
-      {account.address && (
-        <div className="flex justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
-          <a
-            href={`${explorerUrl}account/${
-              account.address
-            }?network=${getChainName(account.chainId)}`}
-            target="_blank">
-            {`${account.address}`}
-          </a>
+    <>
+      {/* Blance */}
+      <div className="flex flex-col items-start m-[0px_16px_16px_16px]">
+        {/* Blance */}
+        <div className="font-[Inter] font-[400] text-[16px] text-[#000000] m-[0px_0px_8px_0px]">
+          Balance
         </div>
-      )}
-
-      {/* Display the Account balance */}
-      <div className="flex justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
-        Balance: {formatEther(account.balance)}
+        {/* (amount) */}
+        <div className="font-[Inter] font-[400] text-[48px] text-[#000000] whitespace-nowrap leading-[58.09px]">{`$ ${number.formatUnitsToFixed(
+          account.balance,
+          18,
+          2
+        )}`}</div>
       </div>
-
-      {/* Show the send button for switching to the Send page */}
-      <div className="flex-col justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
-        <Link href={Path.Send}>Send ↗</Link>
+      {/* action */}
+      <div className="flex justify-evenly m-[0px_0px_24px_0px]">
+        {/* Send */}
+        <Link
+          className="w-[171px] flex items-center rounded-[99px] border-[1px] border-solid border-black"
+          href={Path.Send}>
+          <Up className="w-[24px] h-[24px] m-[16px_2.5px_16px_49px]" />
+          <div className="font-[Inter] font-[400] text-[16px] text-[#000000] m-[18.5px_49px_18.5px_0px]">
+            Send
+          </div>
+        </Link>
+        {/* Receive */}
+        <button className="w-[171px] flex items-center rounded-[99px] border-[1px] border-solid border-black">
+          <Down className="w-[24px] h-[24px] m-[16px_2.5px_16px_38.5px]" />
+          <div className="font-[Inter] font-[400] text-[16px] text-[#000000] m-[18.5px_38.5px_18.5px_0px]">
+            Receive
+          </div>
+        </button>
       </div>
 
-      {/* Display the navigation bar of the Info page */}
+      <Rectangle1 className="w-[390px] h-[1px]" />
```

Comment:
I don't think we need image to play the divider. We can just use css to accomplish it.